java/query/src/test/java/sleeper/query/lambda/SqsQueryProcessorLambdaIT.java

```diff
@@ -640,20 +657,19 @@ public void shouldPublishMultipleStatusReportsToWebSocketForSubQueries() throws
         try {
             // Process Query
             processQuery(query);
-            processQueriesFromQueue(1);
-            processQueriesFromQueue(1);
+            processLeafPartitionQuery(6);
 
             // Then
             wireMockServer.verify(1, postRequestedFor(url).withRequestBody(
                     matchingJsonPath("$.queryId", equalTo("abc"))
                             .and(matchingJsonPath("$.message", equalTo("subqueries")))
                             .and(matchingJsonPath("$.queryIds"))
             ));
-            wireMockServer.verify(1, postRequestedFor(url).withRequestBody(
+            wireMockServer.verify(2, postRequestedFor(url).withRequestBody(
                     matchingJsonPath("$.message", equalTo("completed"))
                             .and(matchingJsonPath("$.recordCount", equalTo("3")))
             ));
-            wireMockServer.verify(1, postRequestedFor(url).withRequestBody(
+            wireMockServer.verify(2, postRequestedFor(url).withRequestBody(
                     matchingJsonPath("$.message", equalTo("completed"))
                             .and(matchingJsonPath("$.recordCount", equalTo("25")))
             ));
```

Comment:
I'm not sure why this is different now. Were there extra requests before that weren't asserted on?

Reply:
I think its something related to the leaf partition queries but not sure where it has come from.

Reply:
It looks like this is a real bug, see my other comment.